tests/sem_version_bionic.bats

```diff
@@ -168,6 +168,8 @@ setup() {
   sem-version node 12.16.1
   run node --version
   assert_line --partial "v12.16.1"
```

Comment:
It would be better to compare latest and actual version of npm if possible, it a new npm will be released test will fail.

Reply:
It can be done with the `npm view npm version`

Reply:
Nice